.github/workflows/site.yml

```diff
@@ -0,0 +1,39 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy static content to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["master"]
```

Comment:
I think it should be "main" instead of "master" - not sure why it even works now.